proposals/exception-handling/Exceptions.md

```diff
@@ -452,42 +444,42 @@ So the list of all sections will be:
 | Function | `3` | Function declarations |
 | Table | `4` | Indirect function table and other tables |
 | Memory | `5` | Memory attributes |
-| Event | `13` | Event declarations |
+| Tag | `13` | Tag declarations |
 | Global | `6` | Global declarations |
 | Export | `7` | Exports |
 | Start | `8` | Start function declaration |
 | Element | `9` | Elements section |
 | Code | `10` | Function bodies (code) |
 | Data | `11` | Data segments |
 
-The event section declares a list of event types as follows:
+The tag section declares a list of tag types as follows:
 
 | Field | Type | Description |
 |-------|------|-------------|
-| count | `varuint32` | count of the number of event to follow |
-| type | `event_type*` | The definitions of the event types |
+| count | `varuint32` | count of the number of tags to follow |
+| type | `tag_type*` | The definitions of the tag types |
```

Comment:
Is `tag_type` defined anywhere?

Reply:
While creating this PR, I found multiple occurrences that we had missed when we had replaced 'exception' with 'event' before, and I tried to replace 'event' with 'tag' in this PR, so I missed those remaining 'exception's.. It was written as `exception_type`. I changed it to `tag_type` now.